docker-compose.yaml

```diff
@@ -13,6 +13,11 @@ services:
       VAULT_TOKEN: "TEST"
       MYSQL_URL: "vault:vault@tcp(mysql:3306)/"
 
+  rabbit:
```

Comment:
Since CircleCI isn't currently running the RMQ tests, we can remove this and leave it out for now

Reply:
done